sources/models/shared-defaults/cf-signal.toml

```diff
@@ -0,0 +1,13 @@
+[settings.cloudformation]
+should-signal = false
+stack-name = ""
+logical-resource-id = ""
+
+[services.cfsignal]
+configuration-files = ["cfsignal-toml"]
+restart-commands = ["/bin/systemctl try-restart cfsignal.service"]
```

Comment:
Do you need a restart command, given that the service should run only once? I might be wrong but in other services that require a reload after a setting change in the API like the docker daemon we have the `restart-command` to actually trigger a realod. But for this service, since there isn't any `affected-services` setting, do you need it at all? Could it be empty?

Reply:
See https://github.com/bottlerocket-os/bottlerocket/pull/1728#discussion_r712282307

Reply:
I'll add the `metadata.settings.cloudformation` `affected-services`.